integration/mediation-tests/tests-transport/src/test/resources/testng.xml

```diff
@@ -23,8 +23,16 @@
 
     <test name="ViewPopRedirectTests" preserve-order="true" verbose="2">
         <classes>
-            <class name="org.wso2.carbon.esb.jms.ViewPopRedirectTests.RedirectTest"/>
-            <class name="org.wso2.carbon.esb.jms.ViewPopRedirectTests.ViewPopTest"/>
+            <class name="org.wso2.carbon.esb.jms.ViewPopRedirectTests.RedirectTest">
+                <methods>
+                    <exclude name=".*"/>
+                </methods>
+            </class>
+            <class name="org.wso2.carbon.esb.jms.ViewPopRedirectTests.ViewPopTest">
+                <methods>
+                    <exclude name=".*"/>
+                </methods>
+            </class>
```

Comment:
Shall we create a GitHub issue to track the removal of this test case?

Reply:
Already added in https://github.com/wso2/micro-integrator/issues/366